test/python/notebooks/test_backend_tools.ipynb

```diff
@@ -27,7 +27,7 @@
     {
       "cell_type": "code",
       "source": [
-        "IBMQ.load_accounts()"
+        "p = IBMQ.load_accounts()"
```

Comment:
```suggestion
        "p = IBMQ.load_account()"
```

Reply:
whoops, that was supposed to be the whole point of this PR 😭